packages/geometry/src/matrix2.ts

```diff
@@ -0,0 +1,59 @@
+import * as Point from './point';
+
+/**
+ * A 2x2 matrix
+ */
+export interface Matrix {
+  a: number;
+  b: number;
+  c: number;
+  d: number;
+}
```

Comment:
Please add some documentation about the where each value is positioned in the matrix. See https://github.com/Vertexvis/vertex-web-sdk/blob/master/packages/geometry/src/matrix.ts#L10-L14

Reply:
Updated. Also updated that documentation at https://github.com/Vertexvis/vertex-web-sdk/blob/master/packages/geometry/src/matrix.ts#L10-L14. Looked like it referenced `b` twice in the doc